src/vcModals.cpp

```diff
@@ -563,6 +564,35 @@ void vcModals_DrawImageViewer(vcState *pProgramState)
   }
 }
 
+void vcModals_DrawProfile(vcState* pProgramState)
+{
+  const char* userName = pProgramState->settings.loginInfo.username;
+  const char* profile = vcString::Get("menuProfileTitle");
```

Comment:
To match the coding standard these should be formatted as 
const char *pUserName 
const char *pProfile